apps/explorer/src/pages/AddressInfoPage/NFTList.tsx

```diff
@@ -144,22 +144,14 @@ interface NFTItemProps {
   onClick: () => void
 }
 
-const NFTItem = ({ nft, onClick }: NFTItemProps) => {
-  const { t } = useTranslation()
-
-  return (
-    <NFTCardStyled onClick={onClick}>
-      <NFTPictureContainer>
-        <NFTThumbnail src={nft.file?.image} showPlayIconIfVideo playOnHover />
-      </NFTPictureContainer>
-      {nft.file?.name ? (
-        <NFTName>{nft.file.name}</NFTName>
-      ) : (
-        <MissingMetadataText>{t('Wrong/old format')}</MissingMetadataText>
```

Comment:
What happens now when there are wrong or old format? The promise rejects, and then?

Reply:
I see now

<img width="1286" alt="image" src="https://github.com/user-attachments/assets/35571204-2303-466c-b543-9b9886d3146e">
<img width="651" alt="image" src="https://github.com/user-attachments/assets/b1901e99-9b17-41ae-b38c-96c0a70f58e8">

all good. But maybe we could still extract some info even if not all of them pass the check. We could probably get the name of the NFT so that the user knows which one we refer to. WDYT?

Reply:
http://localhost:5173/addresses/157iTPFJbe7FqaYGcQmomxqWY9G4n57945eERb6Zt6Ukw

Reply:
nvm, the particular case above is because the ipfs seems to return an error.

In the future, once all apps are with Tanstack, we should merge all different ways of fetching and validating nft data.

Consider this discussion resolved ✅